### DISCUSSION TITLE
Zabbix 6.0 - Media type test failed

### DISCUSSION THREAD

**Discussion by V-Vesely** [AsamK/signal-cli#1514] [Q&A]
Hi,
please I have trouble with sending of message from Zabbix 6.0 via Signal (13.3) . Mannualy message via signal.sh works (I receive it) but from Zabbix when I Test media type "Signal" - test failed: "Failed to read local accounts list."
Permissions for signal.sh are: -r-xr-xr-x zabbix zabbix  

Please did you meet with similar trouble?

Thank you

**Comment by spezialist1** [comment]
Dear @V-Vesely,
Not everything is so simple, because you need to clearly configure _signal-cli_ in the _zabbix_ user environment, that is, in the user environment that will actually start the process of sending the message.  See more details here: https://blog.zabbix.com/getting-your-notifications-via-signal/13286/

**Comment by V-Vesely** [comment]
Hi 
thank you for reaction. I did all steps in this manual but

when permissions for signal.sh are set as in the manual:

![image](https://github.com/AsamK/signal-cli/assets/116262771/5b8ec35b-4517-4f0d-85a6-f5003dc78ff5)

in Zabbix Media type „Signal“  test return me this error message:
![image](https://github.com/AsamK/signal-cli/assets/116262771/e9f4a538-5b23-45f8-afe8-d1032044c524)


when I set permission by this way I got other error message:
![image](https://github.com/AsamK/signal-cli/assets/116262771/ec199bd9-ad62-4d98-af3a-0ef77fa6d148)
![image](https://github.com/AsamK/signal-cli/assets/116262771/0e3aab9e-21dd-4c98-b71a-7210ec9a02bb)

When I send test message via signal.sh directly from linux command line – I receive message without problem

Now I don´t know when I did the failure


Thank you for your time

Vlada Vesely

**Comment by spezialist1** [comment]
Dear @V-Vesely,
Again, you need to make sure that there is a hidden subdirectory with the correct settings for _signal-cli_ in the _/var/lib/zabbix_ home directory, which should belong to the _zabbix_ user (see _/etc/passwd_).  Also, you must run all _signal-cli_ checks in the console as the _zabbix_ user, not as root or some other user.
Can you show how you would run the _signal-cli_ test in the console?

**Reply by V-Vesely** [reply]
Hi,
Iam login to console as user: user
![image](https://github.com/AsamK/signal-cli/assets/116262771/e83541c3-4da7-4f3e-a7b8-22cc54b4a9a0)

/etc/passwd :
![image](https://github.com/AsamK/signal-cli/assets/116262771/e3da47ec-fe54-4e2d-adaf-b7cd54930dd3)

Is my situation better to see?

Thank you
Vlada Vesely

**Reply by spezialist1** [reply]
Dear @V-Vesely,
No, it won't work like that.  The initial registration and other settings of _signal-cli_ must be run on behalf of the _zabbix_ user, i.e. the code command must start like this:
`sudo su - zabbix signal-cli...`

Only after that, subdirectories and configuration files for _signal-cli_ will be automatically created in the _/var/lib/zabbix_ home directory, information from which will be used for further sending or receiving messages.

**Reply by V-Vesely** [reply]
Hello,
it works now - it was enough create home directory for zabbix user ...
Thank you a lot for your help
Vlada Vesely

**Reply by spezialist1** [reply]
I was glad to help you :-).
